modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -162,6 +162,12 @@ def to_numpy(self, **kwargs):
         """
         return self.apply(lambda df, **kwargs: df.to_numpy(**kwargs)).get()
 
+    _iloc = staticmethod(
+        lambda df, row_labels, col_labels: df.iloc[  # noqa: E731 (lambda assignment)
+            row_labels, col_labels
+        ]
+    )
+
```

Comment:
```suggestion
    @staticmethod
    def _iloc(df, row_labels, col_labels):
        return df.iloc[row_labels, col_labels]
```
looks shorter and cleaner 😉